src/main/java/service/impl/WriteToCsvFileImpl.java

```diff
@@ -0,0 +1,30 @@
+package service.impl;
+
+import db.FruitsStorage;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import model.OutFileStructure;
+import service.WriteToFile;
+
+/*
+This class writes all data from the Fruits Storage to the file.
+- receive String filePath, OutFileStructure, FruitsStorage
+ */
+public class WriteToCsvFileImpl implements WriteToFile {
+    @Override
+    public void writeToCsvFile(String filePath, OutFileStructure structure, FruitsStorage storage) {
+        File file = new File(filePath);
+        try (BufferedWriter bufferedWriter = new BufferedWriter(new FileWriter(file, false))) {
+            bufferedWriter.write(structure.getColumn1() + ","
+                    + structure.getColumn2() + System.lineSeparator());
+            for (var node : storage.getFruitsStorage().entrySet()) {
+                bufferedWriter.write(node.getKey() + ","
```

Comment:
All magic numbers, strings, symbols in your code should be constants.
[this article](https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names)

Reply:
Ok Ok ))